app/src/main/java/org/mozilla/fenix/gleanplumb/NimbusMessagingStorage.kt

```diff
@@ -137,20 +137,26 @@ class NimbusMessagingStorage(
     internal fun sanitizeAction(
         messageId: String,
         unsafeAction: String,
-        nimbusActions: Map<String, String>
+        nimbusActions: Map<String, String>,
+        isControl: Boolean
     ): String? {
-        return if (unsafeAction.startsWith("http")) {
-            unsafeAction
-        } else {
-            val safeAction = nimbusActions[unsafeAction]
-            if (safeAction.isNullOrBlank() || safeAction.isEmpty()) {
-                if (!malFormedMap.containsKey(unsafeAction)) {
-                    reportMalformedMessage(messageId)
+
+        return when {
+            unsafeAction.startsWith("http") -> {
+                unsafeAction
+            }
+            isControl -> ""
```

Comment:
Idle nit: is there anything more useful than the empty string you could use? e.g. for `logger.info` purposes, using `NOOP` or `PLACEBO` would more clearly signify a control message.

I don't know the answer, so feel free to disregard.

Reply:
👍🏽 
I'll will go with "CONTROL_ACTION"